src/main/webapp/script.js

```diff
@@ -10,4 +10,23 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License
\ No newline at end of file
+// limitations under the License
+function requestTranslation() {
+    const text = document.getElementById('text').value;
+    const languageCode = document.getElementById('language').value;
```

Comment:
Also, the only other thing is the whitespace. It should be two "spaces" rather than four

Reply:
gotcha